scripts/api-compatibility-check/.baseapi

```diff
@@ -1 +1 @@
-81618929724d94fcc64d4be24caed15357e98446
+v10.3.0
```

Comment:
This was lower before, right? 10.1 or something like that?

Reply:
We have to use the latest public release every time. 
I would like to automate this part this week.